*** ./smali/com/android/server/SystemServer.smali	2016-11-28 06:50:01.789245861 +0000
--- /home/klozz/patchrom/ghost/temp/new_smali/services.jar.out/./smali/com/android/server/SystemServer.smali	2016-11-28 06:51:12.765174335 +0000
***************
*** 142,147 ****
--- 142,149 ----
      .param p0, "args"    # [Ljava/lang/String;
  
      .prologue
+     invoke-static {}, Lmiui/patchrom/ClassHook;->initServerHook()V
+ 
      new-instance v0, Lcom/android/server/SystemServer;
  
      invoke-direct {v0}, Lcom/android/server/SystemServer;-><init>()V
